quantum/process_keycode/process_leader.h

```diff
@@ -32,7 +32,7 @@ void qk_leader_start(void);
 #define SEQ_FOUR_KEYS(key1, key2, key3, key4) if (leader_sequence[0] == (key1) && leader_sequence[1] == (key2) && leader_sequence[2] == (key3) && leader_sequence[3] == (key4) && leader_sequence[4] == 0)
 #define SEQ_FIVE_KEYS(key1, key2, key3, key4, key5) if (leader_sequence[0] == (key1) && leader_sequence[1] == (key2) && leader_sequence[2] == (key3) && leader_sequence[3] == (key4) && leader_sequence[4] == (key5))
 
-#define LEADER_EXTERNS() extern bool leading; extern uint16_t leader_time; extern uint16_t leader_sequence[5]; extern uint8_t leader_sequence_size
```

Comment:
why is this a 16bit value now? It should just be an 8, it only goes up to 5.

Reply:
it wasn't.  `leader_sequence` is what stores the keycodes.  `leader_sequence_size` is what counts the number of entries in the sequence.

Reply:
the counter is separate from the storage.